### PR TITLE
Partners Coverage Testing

### PR DESCRIPTION
**Pull request by reinbach**
Increasing the testing coverage of partners application

Fixes https://github.com/unicef/etools-issues/issues/828

### CODE REVIEW

**Comment by caktus-philip** [comment]
Somewhat related -- `get_agreement_path()` (and possibly other sibling functions) contains a buglet. One of its path components is `agreement_number`, and `agreement_number` sometimes begins with a slash (like ‘/MOU2017844’). When that’s joined into a path, the path contains multiple slashes like in this example (after ‘agreements’) —

'test/file_attachments/partner_organization/935/agreements//PCA2017842/hello_world.txt'

I don't think it breaks anything because UNIX just elides the duplicate path separators, but it makes for some funky path names.

**Comment by reinbach** [comment]
Ha, yes that is not pretty. Thanks, I'll will take a look. `os.path.join` appears to do worse things in this instance! Discards all previous components.

**Comment by robertavram** [comment]
@reinbach we definitely need to keep the logic for now...

**Comment by reinbach** [comment]
@robertavram ok thanks, that's what I needed to know. I'm going to cancel this pull request and reset the code logic changes I made. And stick to just adding testing coverage for now then.

**Comment by robertavram** [comment]
@reinbach hold... :) something's definitely wrong with the code..
we are trying to figure it out

File: EquiTrack/partners/models.py [hunk 1]
```diff
@@ -535,23 +528,20 @@ def planned_cash_transfers(cls, partner, budget_record=None):
...
-                    intervention__status__in=[Intervention.ACTIVE, Intervention.CLOSED, Intervention.ENDED]).exclude(
-                    id=pv_intervention.id).aggregate(models.Sum('programmatic'))['programmatic__sum'] or 0
-                pv += pv_intervention.programmatic
-            else:
-                pv = InterventionPlannedVisits.objects.filter(
-                    intervention__agreement__partner=partner, year=year,
-                    intervention__status__in=[Intervention.ACTIVE, Intervention.CLOSED, Intervention.ENDED]).aggregate(
-                    models.Sum('programmatic'))['programmatic__sum'] or 0
+            pv = InterventionPlannedVisits.objects.filter(
+                intervention__agreement__partner=partner, year=year,
```
Comment (robertavram) [inline EquiTrack/partners/models.py]: 👍

**Pull request closed without merging**